projects/js-packages/components/package.json

```diff
@@ -20,8 +20,8 @@
 		"@automattic/jetpack-base-styles": "workspace:* || ^0.3",
 		"@babel/core": "7.17.9",
 		"@babel/preset-react": "7.16.7",
+		"@testing-library/react": "11.2.7",
```

Comment:
Don't think we need to add it here. We already provide it here: https://github.com/Automattic/jetpack/blob/master/tools/js-test-runner/package.json#L18-L21

Reply:
TS for IDEs won't be happy with that for TypeScript files.
<img width="1115" alt="Screenshot 2022-04-19 at 11 42 30 AM" src="https://user-images.githubusercontent.com/18226415/163937784-6354f542-5df8-40d7-9ded-89c2422177d9.png">

Reply:
Alright, I have added `tsconfig.json` to `js-packages` directory (https://github.com/Automattic/jetpack/pull/23889/commits/3f545911595d7dbafec54d95977d072c128ad94b) to allow imports for dependencies defined in `/tools/*` directories for all the packages inside `js-packages`.

Also, I had to update project structure linting script to allow tsconfig file to reside in `js-packages` - https://github.com/Automattic/jetpack/pull/23889/commits/99bc8529a2e825aabfe343f38c668031de59d581

If those changes are not OK, I can merge `js-packages/tsconfig.json` with `js-packages/components/jsconfig.json` and remove `js-packages/tsconfig.json`.